MODULE.bazel

```diff
@@ -16,7 +16,7 @@
 
 module(
     name = "rules_pkl",
-    version = "0.5.0",
+    version = "0.6.0",
```

Comment:
Please update the version in use in `README.md` as well.

Reply:
Done!